pandas/tests/scalar/test_timestamp.py

```diff
@@ -1106,8 +1106,8 @@ def test_timestamp(self):
 
```

Comment:
Could you please find out if comment from line 1097 is valid after your changes?

Reply:
I'm not sure TBH, I find these comments to be somewhat obscure.